src/demo.js

```diff
@@ -75,9 +58,8 @@ export default class Demo extends Component {
     constructor() {
         super();
         const externals = {
-            user_name: "Anne",
-            user_gender: "female",
-            photo_count: 3,
+            user: "Anne",
+            "emails-count": 3,
```

Comment:
Using dashes in external names is a bad practice. Let's change this back to underscore.